apps/frontend/src/libs/helpers/is-data-loading.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { DataStatus } from "~/libs/enums/enums.js";
+import { type ValueOf } from "~/libs/types/types.js";
+
+const isDataLoading = (dataStatus: ValueOf<typeof DataStatus>): boolean =>
```

Comment:
I would prefer not to add this helper, because sometimes we only need to check for idle or pending state, so this may vary too much